components/cards/EventCard.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <nuxt-link
+    :to="to"
+    class="card-link"
+  >
+    <article class="event-card">
+      <div class="event-card__content">
+        <header>
+          <span class="event-card__subtitle">
+            {{ type }}
+          </span>
+          <h3 class="event-card__title">
+            {{ title }}
+          </h3>
+        </header>
+
+        <footer>
+          <div class="event-card__info-detail">
+            <Map20 class="event-card__icon-map" />
+            <span class="event-card__place">{{ place }}</span>
+          </div>
+          <div class="event-card__date-and-arrow">
+            <div class="event-card__info-detail">
+              <Calendar20 class="event-card__icon-calendar" />
+              <span class="event-card__date">{{ date }}</span>
+            </div>
+            <ArrowRight20 class="event-card__icon-arrow-right" />
+          </div>
+        </footer>
+      </div>
+
+      <div
+        class="event-card__media"
+        :style="`background-image: ${decorate(image)};`"
+      />
+    </article>
+  </nuxt-link>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {
+  @Prop(String) type
+  @Prop(String) title
+  @Prop(String) image
+  @Prop(String) place
+  @Prop(String) date
+  @Prop(String) to
+
+  decorate (image) {
+    const bgEffects = [
+      `url(${image})`
+    ]
+    return bgEffects.join(', ')
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@mixin config-icons($prefix, $colors...) {
+  @each $color in $colors {
+    .#{$prefix}#{nth($color, 1)} {
+      fill: nth($color, 2);
+      position: relative;
+      top: .3em;
+      width: 1.25rem;
+      height: 1.25rem;
+    }
+  }
+}
+
+@include config-icons('event-card__icon-',
+    'map' var(--white),
+    'calendar' var(--white),
+    'arrow-right' var(--purple-60)
+);
+
+.card-link {
+  text-decoration: none;
+}
+
+.event-card {
+  height: 15.875rem;
```

Comment:
Try to use only 2 decimals on rems. 3 are too much and you don't need to be that precise.